### DISCUSSION TITLE
Release 3.3

### DISCUSSION THREAD

**Discussion by revoxhere** [revoxhere/duino-coin#1550] [General]
## Three years of Duino-Coin (part 1?)
Hello miners, this August is a bit special - three years ago first versions of Duino-Coin were born. I never anticipated such a big success of my side project, but I'm very happy how things turned (and are turning) out. The Duino team is amazing, just like the community that gathered around the coin. There are thousands of devices mining DUCO, there are articles about us on various websites and magazines, and the best thing is that really enjoy developing and expanding my project 😍 <br>
I'd like to thank everyone for beliving in us, I hope next three years will be even more dynamic than the previous ones and we will expand even further 💯 <br>
Later this month we will be organizing some giveaways, events and other cool stuff as a celebration 😎 🥳  

## What's new, what's changed 👀 
* **Improved ESP8266 performance by @snakeye in https://github.com/revoxhere/duino-coin/pull/1530**
* **Improved Raspberry Pi Pico (and similar boards) support in the AVR Miner (3.25) by @revoxhere**
* Updated Spanish README by @Omicron166 in https://github.com/revoxhere/duino-coin/pull/1531
* Updated Czech translation by @fpetru104 in https://github.com/revoxhere/duino-coin/pull/1546
* Updated the Minimal PC Miner by @Omicron166 in https://github.com/revoxhere/duino-coin/pull/1532
* Fixed mining key related bugs by @Omicron166 in https://github.com/revoxhere/duino-coin/pull/1528
* Fixed debug_output error by @miqueiaspenha in https://github.com/revoxhere/duino-coin/pull/1539

### Node-side updates 💾 
* **Added adaptive ping compensation by @revoxhere**
* **Added new node: coffe-pool-1 hosted by @revoxhere**
* Added missing newlines in messages that slowed down ESP32 mining by @revoxhere 
* Added some Kolka tweaks and adjustments for better rewards by @revoxhere 
* Fixed miner count/kolka identifier for dual core ESP32s by @revoxhere 
* Updated difficulty tiers to work with the new ESP8266 code by @revoxhere
* Updated Raspberry Pi Pico support by @revoxhere 

## New contributors 👋 
* @miqueiaspenha made their first contribution in https://github.com/revoxhere/duino-coin/pull/1539
* @snakeye made their first contribution in https://github.com/revoxhere/duino-coin/pull/1530

### ⚠️ NOTICE: .exe binaries may be detected as a virus by your antivirus software. This is a **false positive** caused by pyinstallers bootloader, read about it [here](https://stackoverflow.com/questions/43777106/program-made-with-pyinstaller-now-seen-as-a-trojan-horse-by-avg). Duino-Coin is not a virus.

**Full Changelog**: https://github.com/revoxhere/duino-coin/compare/3.2...3.3
As always, please report any encountered issues. Happy mining!

<hr /><em>This discussion was created from the release <a href='https://github.com/revoxhere/duino-coin/releases/tag/3.3'>Release 3.3</a>.</em>

**Comment by CosmicShreyas** [comment]
Hello Revox may i also join the project once again and help? (not related to siricoin or anything im not taking or promoting my stuff here this is all personal)

**Reply by iqnbet** [reply]
Good afternoon. I can't register a wallet gives an error. example below, what to do?


Error

An error has ocurred.
There may be issues with the servers. Please try again later.

If you are still having problems, please contact us on Discord with the following data:


Error: TypeError: Failed to fetch
